core/sis-utility/src/main/java/org/apache/sis/util/collection/WeakHashSet.java

```diff
@@ -146,7 +146,6 @@ public WeakHashSet(final Class<E> type) {
         @SuppressWarnings("unchecked")
         @Workaround(library="JDK", version="1.7")
         final Entry[] table = (Entry[]) Array.newInstance(Entry.class, MIN_CAPACITY);
-//      table = new Entry[size];
```

Comment:
Just before this comment, they were another comment explaining why we keep this comment.

Reply:
Missed that comment.
Going to undo.